packages/react-components/src/components/RichTextEditor/Plugins/PlaceholderPlugin.tsx

```diff
@@ -50,11 +41,4 @@ export class PlaceholderPlugin extends WatermarkPlugin {
     super.hide(editor);
     this.isPlaceholderShown = false;
   }
-
-  private onCompositionStart = (): void => {
```

Comment:
Updated the Watermark plugin as the same code is added to rooster plugin: https://github.com/microsoft/roosterjs/blob/54c7dbe316cc2a6ccc986490390a2dcc954dfece/packages/roosterjs-content-model-plugins/lib/watermark/WatermarkPlugin.ts#L108